### PR TITLE
#17 Fix tweet text && Added pipenv

### PR DESCRIPTION
**Pull request by MinoLiu**
line 264 275
```python
        # Retrieving original content from retweet
        if 'retweeted_status' in obj:
            if 'extended_tweet' in obj['retweeted_status']:
                tweet['text'] = obj['retweeted_status']['extended_tweet']['full_text']
            else:
                tweet['text'] = obj['retweeted_status']['text']
        # Retrieving full text from extended_tweet
        elif 'extended_tweet' in obj:
            tweet['text'] = obj['extended_tweet']['full_text']

        else:
            tweet['text'] = obj['text']
```

[Pipenv](https://github.com/pypa/pipenv) Usage
```sh
pipenv install --dev
pipenv run python streamtest.py
```
And
`variable != None` => `variable is not None`
`variable == None` => `variable is None`

### CODE REVIEW

**Comment by MinoLiu** [comment]
If there is `retweet_status` this tweet is retweet.
If there is `extended_tweet`, the full_text of this tweet will be in extended_tweet

File: streamtest.py [hunk 1]
```diff
@@ -12,314 +12,315 @@
...
+    ans = get_tweets_from_db()
+    si = io.StringIO()
+    fieldnames = [
+        'retweet_count', 'user_friends_count', 'created_at', 'user_followers_count', 'reply_count', 'name', 'location',
+        'keyword', 'favorite_count', 'user_time_zone', 'tweet_hashtags', 'lang', 'user_id', 'text', 'user_description',
+        'screen_name', 'retweeted', 'timestamp_ms', '_id', 'tweet_text_urls', 'url'
+    ]
+    writer = csv.DictWriter(si, fieldnames=fieldnames)
+    writer.writeheader()
+    writer.writerows(si)
```
Comment (MinoLiu) [inline streamtest.py:236]: This was originally s, but there is no such variable so I changed it to si

File: streamtest.py [hunk 2]
```diff
@@ -12,314 +12,315 @@
...
+    ans['tweets'] = json_util._json_convert(s)
+    return jsonify(ans)
+
+
+# --------------Return Tweets--------------------
 @app.route('/gettweets', methods=['GET'])
 def getTweets():
-	ans=get_tweets_from_db()
-	return jsonify(ans)
+    return get_tweets_from_db()
```
Comment (MinoLiu) [inline streamtest.py:219]: Cause get_tweets_from_db returns already jsonify so I delete it.

**Comment by shreybatra** [comment]
@Sean2525 great work! Just one ques, why do we need pip life and piplock file?

**Comment by MinoLiu** [comment]
@shreybatra 
Generated by pipenv, piplock is used to control the version
Pipfile is used to collect installed packages.
For details, please see https://github.com/pypa/pipenv

**Comment by shreybatra** [comment]
@Sean2525 why are we restricting the version?

**Comment by MinoLiu** [comment]
```env
[[source]]
url = "https://pypi.org/simple"
verify_ssl = true
name = "pypi"

[packages]
tweepy = "*"
pymongo = "*"
flask = "*"

[dev-packages]
pylint = "*"
"flake8" = "*"
yapf = "*"

[requires]
python_version = "3.6"
```
In `Pipfile` **\*** allows the upgrade, so we did not restrict the version of packages. 
And the `Pipfile.lock` is used to make sure **dependencies** version of packages are same.
If didn't need it, just add in gitignore.

**Pull request closed without merging**